packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -490,6 +474,40 @@ export class GrpcPlugin extends BasePlugin {
   static convertGrpcStatusToSpanStatus(statusCode: grpcTypes.status): number {
     return statusCode;
   }
+
+  /**
+   * Returns a span context on a Metadata object if it exists and is
+   * well-formed, or null otherwise.
+   * @param metadata The Metadata object from which span context should be
+   * retrieved.
```

Comment:
Nit: could you indent "retrieved" by 4 spaces to show it is a continuation of the `@param` JSDoc tag?